Assets/MixedRealityToolkit.Extensions/SpectatorView/Scripts/Compositor/ICalibrationParser.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Extensions.Experimental.SpectatorView.Compositor
+{
+    /// <summary>
+    /// Interface to allow multiple parsers to attempt to parse calibration data.
+    /// </summary>
+    public interface ICalibrationParser
+    {
+        /// <summary>
+        /// Tries to parse the provided string as calibration data.
+        /// </summary>
+        /// <param name="rawCalibrationData">The string representation of calibration data to deserialize.</param>
+        /// <param name="calibrationData">The parsed calibration data, which can be used to configure a stereo camera rig.</param>
+        /// <returns>True if the string was successfully parsed, otherwise false.</returns>
+        bool TryParse(string rawCalibrationData, out ICalibrationData calibrationData);
```

Comment:
>string [](start = 22, length = 6)

I'm not sure that we should assume that calibration data will be handed around in a string. Would it be possible to make this a byte array instead?

Reply:
For the calibration data to be JSON, it either needs to always be a string, or be stored in the JSON blob as a byte[].  Right now I've opted for a string but we can revisit this later.